build.sbt

```diff
@@ -23,9 +23,9 @@ publishArtifact in Test := false
 pomIncludeRepository := { x => false }
 
 libraryDependencies ++= Seq(
-  "io.swagger" % "swagger-core" % "1.5.8",
-  "org.scalatest" %% "scalatest" % "2.2.5" % "test",
-  "com.fasterxml.jackson.module" %% "jackson-module-scala" % "2.4.2",
```

Comment:
Thanks @siroccj.  Jackson 2.8 has some more changes so 2.7 seems like a good start. Cowtowncoder recommended a similar approach. I'm away for a few weeks but I can at fixing the issue in my Swagger-core pull request when I get back.